state/state.go

```diff
@@ -130,9 +129,70 @@ func (o *State) Initialize(
 	return nil
 }
 
+// Load loads the state from the database. It will loads the census root,
+// ballot mode, encryption key, results add and results sub from the database.
+// The database and the process ID should be defined before calling this
+// function. If the state is not initialized or something fails to load, an
+// error is returned.
+func (o *State) Load() error {
+	if o.processID == nil || o.db == nil || o.tree == nil {
+		return fmt.Errorf("state not initialized")
+	}
+	// set the process ID
+	o.Process.ID = o.processID
```

Comment:
this looks a bit weird, what is o.processID ? Why do we have it duplicated?

I'd review the state API and ensure we are being idiomatic, `Load(processID)` seems more logic to me.

Reply:
had the same feeling about o.processID when i read this, and this code debt is mine :sweat_smile:

Reply:
not sure how to best solve it, i just agree it looks wrong

Reply:
i pushed aa78907 to fix this